packages/shared/src/components/RichTextEditor/types.ts

```diff
@@ -10,12 +10,9 @@ export type RichTextEditorProps = {
     columns?: number;
     gap?: string;
     plugins?: PluginComposer;
-    updateValueOnChange?: boolean;
     showSerializedText?: boolean;
-    shouldPreventPageLeave?: boolean;
-    onBlur?: (value: string) => void;
-    onTextChange?: (value: string) => void;
-    onValueChanged?: (value: TreeOfNodes | null) => void;
+    updateValueOnChange?: boolean;
+    onTextChange: (value: string) => void;
```

Comment:
Should this really be the only non-optional prop?

Reply:
I see :D. I was just wondering if it ever can be undefined and I guessed "no". Didn't overthink the other ones, but know thinking about it maybe the id should be non-optional, but all others can be optional. Or what do you think?

Reply:
I think we could keep `onTextChange` optional too, then we would not need to adjust all the tests.

Reply:
Changed 👍